.github/workflows/nightly.yml

```diff
@@ -36,6 +36,21 @@ jobs:
           mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -Dprofile=web -DjavaVersion=8 -DoutputDirectory=app -Dgoals="clean package"
           rm -rf app
 
+
+      - name: Run Archetype for EE 8 Core Profile, SE 8
+        continue-on-error: true
+        run: |
+          mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -Dprofile=core -DjavaVersion=8 -DoutputDirectory=app -Dgoals="clean package" | tee mvn_output.txt
+
+      - name: Check Maven Build Status
+        run: |
+          ERROR_MESSAGE="Failed to execute goal org.apache.maven.plugins:maven-archetype-plugin:3.2.1:generate (default-cli) on project jakarta-starter-modules: Failed, the Core Profile is only supported for Jakarta EE 10"
```

Comment:
Can you please shorten the error messages to something like "Core Profile is only supported for Jakarta EE 10"? Otherwise this will be a bit too brittle of a test. The same for the other ones.

Reply:
Could you please clarify why it's necessary to check for the failure status when we are already checking for the error message, which should indicate that the build failed?

Reply:
It's subjective, but the main issue is readability/maintainability. If we explicitly check for failure status, it's semantically clearer to a maintainer what we are testing for. There is also an edge case (less of a concern) that we generate an error message in the Archetype but don't actually fail the build (which in case of Docker is actually valid and what we should check for in the test). Does that help?

Reply:
Yes. Thank you for the clarification.